mslib/msui/wms_control.py

```diff
@@ -27,7 +27,7 @@
 """
```

Comment:
The query string has a fixed format given by the WMS standard. That is not that easy to change as it looks like.
I think it is better to look on this in a seperate Pull Request ot to keep it as it is. A change in the format will make us likly incompatible.